src/core/CharacterState/Walking.lua

```diff
@@ -67,43 +65,35 @@ local function createForces(character)
 	}
 end
 
--- loop between 0 - 2*pi
-local function angleAbs(angle)
-	while angle < 0 do
-		angle = angle + THETA
-	end
-	while angle > THETA do
-		angle = angle - THETA
-	end
-	return angle
-end
-
 local function angleShortest(a0, a1)
-	local d1 = angleAbs(a1 - a0)
-	local d2 = -angleAbs(a0 - a1)
-	return math.abs(d1) > math.abs(d2) and d2 or d1
+	return (a1 - a0 + math.pi)%(2*math.pi) - math.pi
```

Comment:
Ugh. Figured this out before but it was late and I couldn't find it. I am ashamed.